scripts/hail_batch/vep/main.py

```diff
@@ -32,13 +32,19 @@ def main(script: str, mt: str):
 
     job = dataproc.hail_dataproc_job(
         batch=batch,
+        worker_machine_type='n1-highmem-8',
+        worker_boot_disk_size=200,
+        secondary_worker_boot_disk_size=200,
         script=f'{script} --mt {mt}',
         max_age='12h',
-        init=['gs://cpg-reference/hail_dataproc/install_common.sh'],
+        init=[
+            'gs://cpg-reference/hail_dataproc/install_common.sh',
+            'gs://cpg-reference/vep/vep-GRCh38.sh',
+        ],
         job_name='run_vep',
-        num_secondary_workers=20,
+        num_secondary_workers=50,
```

Comment:
You can keep 20 workers. The only requirement is that the number of standard workers (`num_workers`) should be 5-10% of `num_secondary_workers`.